docs/guides/cloud/debugging/test-replay.mdx

```diff
@@ -0,0 +1,109 @@
+
+## Getting Started
+
+:::info
+
+Test Replay is available to **all** Cypress Cloud plans at no additional cost, subject to usage limits. Free plans allow up to 500 test results per month, while exceeding this limit requires upgrading the plan.
+
+:::
+
+All you need to do is log into your Cypress Cloud account without requiring any additional software or changes to the existing test run recording process.
+
+Test Replay is automatically enabled when updating to `v13` of Cypress and an opt out is provided through project-level settings. Learn about what this means for [capturing video](#See-also) in Cypress Cloud below.
+
+## Accessing Test Replay
+
+Accessing the replay of your test is as simple as reviewing any other test artifact in Cypress Cloud.
+
+Hover on a "Test for review" under the Latest Runs Overview.
+
+<DocsImage
+  src="/img/guides/cloud/test-replay/test-overview.jpg"
+  alt="Run overview row"
+/>
+
+Navigate to the [test detail sidebar](/guides/cloud/debugging/recorded-runs#Test-detail-sidebar) header or above each attempt in the errors panel.
+
+<DocsImage src="/img/guides/cloud/test-replay/sidebar.jpg" alt="Run sidebar" />
+
+## Debugging With Test Replay
+
+This displays the Test Replay interface, which we'll unpack from top to bottom in more detail below.
+
+<DocsImage
+  src="/img/guides/cloud/test-replay/replay-demo.gif"
+  alt="Test Replay Demo"
+/>
+
+### Run header
+
+The left side of the header displays the test run # with the Git branch name along with the spec, platform and browser information. Hovering on this exposes additional test meta data.
+
+<DocsImage
+  src="/img/guides/cloud/test-replay/test-replay-header2.jpg"
+  alt="Test Replay header"
+/>
+
+### Command Log
+
+The left side of the screen displays the familiar command log, where you can step through your test just like you would in the Cypress app.
+
+<DocsImage
+  src="/img/guides/cloud/test-replay/command-log.png"
+  alt="Test Replay Developer Tools panel"
+  width="60%"
+/>
+
+### Developer Tools
+
+Dig deeper into all network-level traffic, console logs and inspect the application under test just as you can in Chrome.
+
+<DocsImage
+  src="/img/guides/cloud/test-replay/dev-tools.png"
+  alt="Test Replay Developer Tools panel"
+  width="60%"
+/>
+
+### Replay Controls
+
+Toggle each test attempt, play back the error at four different speeds or scrub through the error by dragging the playhead along the timeline.
+
+<DocsImage
+  src="/img/guides/cloud/test-replay/scrubber.png"
+  alt="Test Replay scrubber controls"
+/>
+
+## Browser Compatability
+
+Test Replay currently supports Chromium-based browsers only. Additionally, Cypress Cloud allows running Cypress for a single browser per group definer. For example, Firefox tests require video recording, while Chrome tests can utilize Test Replay.
```

Comment:
- How does this video recording work with Firefox, considering
https://github.com/cypress-io/cypress/issues/18415?